packages/graphql/src/schema/get-unions.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { UnionTypeDefinitionNode } from "graphql";
+import { Node } from "../classes";
+import { Union } from "../classes/Union";
+
+type Unions = {
+    unions: Union[];
+};
+
+function getUnions(unionTypes: UnionTypeDefinitionNode[], options: { nodes: Node[] }): Unions {
```

Comment:
Why not returning directly `Union[]`?

```suggestion
function getUnions(unionTypes: UnionTypeDefinitionNode[], options: { nodes: Node[] }): Union[] {
```